cli/command/init/cmd.go

```diff
@@ -68,8 +56,47 @@ var (
 	restoTimeout        = 5 * time.Second
 )
 
+// Command creates the `run` command
+func Command(cli *command.SauceCtlCli) *cobra.Command {
+	initCfg := &initConfig{}
+
+	cmd := &cobra.Command{
+		Use:     initUse,
+		Short:   initShort,
+		Long:    initLong,
+		Example: initExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			log.Info().Msg("Start Init Command")
+			err := Run(cmd, initCfg)
+			if err != nil {
+				log.Err(err).Msg("failed to execute init command")
+				sentry.CaptureError(err, sentry.Scope{})
+				os.Exit(1)
+			}
+		},
+	}
+	cmd.Flags().StringVarP(&initCfg.username, "username", "u", "us-west-1", "username to use")
+	cmd.Flags().StringVarP(&initCfg.accessKey, "accessKey", "a", "us-west-1", "username to use")
+	cmd.Flags().StringVarP(&initCfg.region, "region", "r", "us-west-1", "region to use")
+	cmd.Flags().StringVarP(&initCfg.frameworkName, "framework", "f", "", "framework to configure")
+	cmd.Flags().StringVarP(&initCfg.frameworkVersion, "frameworkVersion", "v", "", "framework version to be used")
+	cmd.Flags().StringVarP(&initCfg.cypressJSON, "cypress.config", "", "", "path to cypress.json file (cypress only)")
+	cmd.Flags().StringVarP(&initCfg.app, "app", "", "", "path to application to test (espresso/xcuitest only)")
+	cmd.Flags().StringVarP(&initCfg.testApp, "testApp", "t", "", "path to test application (espresso/xcuitest only)")
+	cmd.Flags().StringVarP(&initCfg.platformName, "platformName", "p", "", "platform name")
+	cmd.Flags().StringVarP(&initCfg.browserName, "browserName", "b", "", "browser name")
+	cmd.Flags().StringVarP(&initCfg.artifactWhenStr, "artifacts.download.when", "", "fail", "defines when to download artifacts")
+	cmd.Flags().StringVarP(&initCfg.device.Name, "device.name", "d", "", "defines which device to target")
+	cmd.Flags().StringVarP(&initCfg.emulator.Name, "emulator.name", "", "", "defines which emulator to target")
+	cmd.Flags().StringArrayVarP(&initCfg.emulator.PlatformVersions, "emulator.platformVersion", "", []string{}, "defines which emulator version to target")
```

Comment:
Would be cool if it was consistent with espresso/xcuit:
https://github.com/saucelabs/saucectl/blob/master/cli/command/run/espresso.go#L71-L72

Reply:
Just updated to make it consistent:

Work correctly this way now:
```
$> go run ./cmd/saucectl/saucectl.go init --framework espresso --emulator 'name=Samsung Galaxy S8 FHD GoogleAPI Emulator,platformVersion=7.1' --app ./tests/e2e/espresso/calc.apk --testApp ./tests/e2e/espresso/calc-success.apk 
10:37:32 INF Start Init Command

The following files have been created:
  .sauce/config.yml

$>  go run ./cmd/saucectl/saucectl.go init --framework espresso --device 'name=HTC .*' --app ./tests/e2e/espresso/calc.apk --testApp ./tests/e2e/espresso/calc-success.apk                                                        
10:39:54 INF Start Init Command

The following files have been created:
  .sauce/config.yml

$>
```